wire-ios/Wire-iOS/Wire-Info.plist

```diff
@@ -171,6 +171,29 @@
 	</array>
 	<key>UTExportedTypeDeclarations</key>
 	<array>
+		<dict>
+			<key>UTTypeConformsTo</key>
+			<array>
+				<string>public.data</string>
+			</array>
+			<key>UTTypeDescription</key>
+			<string>Wire Universal Backup File</string>
```

Comment:
I also chose this name without aligning with other teams.
Also we might want to localize it, if possible.

Reply:
If this is user facing then yes we should check to align it, I'll ask.

Reply:
Call it `Wire Backup File`

Reply:
done: c7d69aac652afcc3b0fc1c61ce563f12c99934bf